### ISSUE TITLE
Redesign my videos list page

### ISSUE THREAD

**Issue by Chocobozzz** [Chocobozzz/PeerTube#5505]
### Describe the problem to be solved

It seems to me that this page is not very practical, because video rows are quite big and don't display a lot of information.

### Describe the solution you would like

I find (and it seems most admins agree) the videos overview table with small thumbnails allows us to display more useful information that are condensed inside a reduced height, so it's easier to have a quick overview of the videos.
Youtube also uses the "table" UX.

I propose to reuse the table UX, quite the same as the admin one to display "My videos".

Unfortunately we won't have enough resources to keep the both kind of views (big rows/table).

**Comment by DalekDave** [comment]
I agree that it would be a good idea on my home page (https://dalek.zone) to have smaller channel and video thumbnails.

I post a channel thumbnail for each active user on my instance, and I would like the layout to be more compact.

On Chrome, you can set the default magnification of a page, and that has produced a better look on Dalek Zone's home page. But it is not supported by Firefox.

Is a table-base approach good for SEO? Maybe it would be enough just to re-work the CSS in the existing implementation?

**Comment by Chocobozzz** [comment]
This issue is about `/my-library/videos` endpoint, not the homepage

**Comment by DalekDave** [comment]
I agree that the 'My videos' page would be better if it were laid out more like the 'Subscriptions' or 'Discover' pages, with the videos being divided into sections based on their category.

IMHO, this would be a good idea for the 'My playlists' and 'My channels' pages as well.

**Comment by Chocobozzz** [comment]
Implemented in https://github.com/Chocobozzz/PeerTube/commit/b295dd582022db49dbd415353be691a47ae6d522
